ext/opentelemetry-ext-cloud-trace/src/opentelemetry/ext/cloud_trace/__init__.py

```diff
@@ -0,0 +1,236 @@
+# Copyright OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Cloud Trace Span Exporter for OpenTelemetry."""
+
+import logging
+from typing import Sequence, Dict, Any, List
+
+import opentelemetry.trace as trace_api
+from google.cloud.trace_v2 import TraceServiceClient
+from google.cloud.trace_v2.proto.trace_pb2 import AttributeValue
+from opentelemetry.sdk.trace import Event
+from opentelemetry.sdk.trace.export import Span, SpanExporter, SpanExportResult
+from opentelemetry.util import types
+
+logger = logging.getLogger(__name__)
+
+# Max length is 128 bytes for a truncatable string.
+MAX_LENGTH = 128
+
+
+class CloudTraceSpanExporter(SpanExporter):
+    """Cloud Trace span exporter for OpenTelemetry.
+
+    Args:
+        client: Cloud Trace client.
+        project_id: project_id to create the Trace client.
+    """
+
+    def __init__(
+            self, project_id, client=None,
+    ):
+        self.client = client or TraceServiceClient()
+        self.project_id = project_id
+
+    def export(self, spans: Sequence[Span]) -> SpanExportResult:
+        """Export the spans to Cloud Trace.
+
+        See: https://cloud.google.com/trace/docs/reference/v2/rest/v2/
+             projects.traces/batchWrite
+
+        Args:
+            spans: Tuple of spans to export
+        """
+        cloud_trace_formatted_spans = self.translate_to_cloud_trace(spans)
```

Comment:
This variable is only used once. Should we just have `for span in self.translate_to_cloud_trace(spans):` below?